server/config/config.go

```diff
@@ -607,6 +611,7 @@ const (
 	// hot region.
 	defaultHotRegionCacheHitsThreshold = 3
 	defaultSchedulerMaxWaitingOperator = 3
+	defaultLeaderScoreStrategy         = "size"
```

Comment:
how about make count default?

Reply:
I am not sure if there will be conflicts with the existing schedule by size when update.

Reply:
I think it's ok to make `count` as default. there are also many transfer leader operators caused by the `evict-leader` when doing a rolling update.